core/imageroot/var/lib/nethserver/node/actions/remove-module/50update

```
@@ -38,6 +38,7 @@ if [[ -z $uid ]]; then # Rootfull module
     systemctl disable --now agent@"${mid}"
     if [[ ${preserve} == "no" ]]; then
         rm -rf "/var/lib/nethserver/${mid}"
+        podman volume rm $(podman volume ls | grep -o "${mid}-\w*")
```

1. Handle the case where grep produces an empty list
2. Ignore any error

```suggestion
         podman volume ls | grep -o "${mid}-\w*" | xargs -r -- podman volume rm || :
```